### PR TITLE
[flutter_tools] add more information to obfuscate option

### PR DESCRIPTION
**Pull request by jonahwilliams**
## Description

Taken directly from https://github.com/dart-lang/sdk/issues/30524#issuecomment-324905442

### CODE REVIEW

File: packages/flutter_tools/lib/src/runner/flutter_command.dart [hunk 1]
```diff
@@ -400,7 +400,11 @@ abstract class FlutterCommand extends Command<void> {
         'mapping between the values and the original identifiers is stored in the '
         'symbol map created in the specified directory. For an app built with this '
         'flag, the \'flutter symbolize\' command with the right program '
-        'symbol file is required to obtain a human readable stack trace.',
+        'symbol file is required to obtain a human readable stack trace.\n\n'
+        'Because all identifiers are renamed methods like Object.runtimeType, '
```
Comment (dnfield) [inline packages/flutter_tools/lib/src/runner/flutter_command.dart]: I think a comma between renamed and methods would make this clearer

**Review by dnfield — APPROVED** [review]
Lgtm w/ nit

File: packages/flutter_tools/lib/src/runner/flutter_command.dart (hunk 1 above)
Comment (jonahwilliams) [reply packages/flutter_tools/lib/src/runner/flutter_command.dart]: I think It  works better If  I remove the `like`

Comment (dnfield) [reply packages/flutter_tools/lib/src/runner/flutter_command.dart]: The part that threw me was that were not talking about renamed methods

Comment (jonahwilliams) [reply packages/flutter_tools/lib/src/runner/flutter_command.dart]: Okay fixed for real

**Pull request merged**